### ISSUE TITLE
Add reader functionality

### ISSUE THREAD

**Issue by tnyeanderson** [jellyfin/jellyfin-plugin-bookshelf#7]
Using this library it should be very simple. I may take some time to work on this in the next few weeks.

https://github.com/futurepress/epub.js

**Comment by dkanada** [comment]
I completely agree we should add this, but it would actually get added ni the web client, not this plugin. I'll transfer the issue over there so we can track this. If you happened to open a pull request for this we would definitely prioritize a merge since it would greatly benefit users.

**Comment by dkanada** [comment]
I completely agree we should add this, but it would actually get added in the web client, not this plugin. I'll transfer the issue over there so we can track this. If you happened to open a pull request for this we would definitely prioritize a merge since it would greatly benefit users.
